crates/vm/levm/src/vm.rs

```diff
@@ -610,37 +615,51 @@ impl VM {
     }
 
     /// Gets the max blob gas cost for a transaction that a user is willing to pay.
-    fn get_max_blob_gas_cost(&self) -> Result<U256, VMError> {
-        let blob_gas_used = U256::from(self.env.tx_blob_hashes.len())
+    fn get_max_blob_gas_price(&self) -> Result<U256, VMError> {
+        let blob_gas_used: u64 = self
```

Comment:
I know you rewrite this but this is not `blob_gas_used`

Reply:
Noted, I changed it to `blob_gas_amout` (https://github.com/lambdaclass/ethrex/pull/1528/commits/e0a5eec01c68aa16348e9deb7743e31c16e58498), what do you think?

Reply:
Well, it is actually not `blob_gas_amount`, in that case it is just the length of the blob hash list! Not related to gas

Reply:
Sorry Jere, it was reading the variable name wrong in my head. It's been a rough day! Haha,

Changed here: https://github.com/lambdaclass/ethrex/pull/1528/commits/341ca8d2ccf93ab7e932dc7fa8b8fdc05d405cf4